recipes/pcst-fast/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pcst-fast" %}
+{% set version = "1.0.10" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://files.pythonhosted.org/packages/d4/fd/64b51c867bad63e6622ff97cad6230b94b19b5a61e30424cd69c8353091c/pcst_fast-{{ version }}.tar.gz
+  sha256: 3b5694110ce2e004471f383267d5e4ab7fe1ba9828954e8c42560ac1e42b25e6
+
+build:
+  number: 0
+  skip: True  # [py < 38]
+  script: {{ PYTHON }} -m pip install . --no-build-isolation --no-deps --use-pep517 -vvv
+  run_exports:
+    - {{ pin_subpackage('pcst-fast', max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+  host:
+    - pip
+    - pybind11 >=2.4
```

Comment:
Upstream specifies `pybind11 >=2.1.0`. Do you know the history of why the recipe bumped this? If 2.4 is required, a PR should be sent to upstream; otherwise, this should be set to match upstream.

Reply:
Eagle-eyed! I don't know of any reason why it was set to >=2.4. The original bioconda submission was set to this. I'll revert to match upstream.